### ISSUE TITLE
Update img or src props not updating the edit area

### ISSUE THREAD

**Issue by alonenascimento** [kirill3333/react-avatar#58]
When i set the state.src with a url or base 64img the edit area is not updated

 ```
axios.get("/images/img.jpg")
      .then( (response) => {
        self.setState({
            src:response.data
        });
        
        
      })
      .catch( (error) => {
        console.log(error);
      });  
```

**Comment by nirav-panchal** [comment]
Yes, I am also facing this issue. Src value is not update or clear after successful form submission. Can you please provide solution for it.

**Comment by kirill3333** [comment]
@alonenascimento @nirav-panchal sorry for the delay it was fixed in 1.1.0 version
